app/models/embed/purl.rb

```diff
@@ -16,11 +16,12 @@ def attributes=(hash)
 
     attr_accessor :druid, :type, :title, :use_and_reproduction, :copyright, :contents, :collections,
                   :license, :bounding_box, :embargo_release_date, :archived_site_url, :external_url,
-                  :embargoed, :stanford_only_unrestricted, :public, :controlled_digital_lending,
+                  :embargoed, :citation_only, :stanford_only_unrestricted, :public, :controlled_digital_lending,
                   :etag, :last_modified
 
     alias embargoed? embargoed
     alias stanford_only_unrestricted? stanford_only_unrestricted
+    alias citation_only? citation_only
```

Comment:
Is this needed?

Reply:
Well, I want to check for citation_only below (here): https://github.com/sul-dlss/sul-embed/pull/1971/files#diff-299fb888af648917064acec2d633ac5ebfa31cdbb3794359a5de888ca27f371fR49

But now I realize I don't have the corresponding citation_only methods coming from both the XML and JSON PURL loaders.  I can easily add one for the JSON parsing, but the XML parsing comes from DorRightsAuth and while they do have a citation_only method, it seems to be used only by the indexer now: https://github.com/sul-dlss/dor-rights-auth/blob/main/lib/dor/rights_auth.rb#L76-L79

pondering how best to do this

Reply:
Why do you need to check for citation only.  Can't you just check the download access on the files?  That will also be no-download if the object is citation only.

Reply:
Ahh, ok...so just don't bother with a citation only check.